src/Telegram.php

```diff
@@ -351,6 +354,15 @@ public function handleGetUpdates($limit = null, $timeout = null)
         }
 
         if ($response->isOk()) {
+            // Set last update_id based on returned updates
+            if (!DB::isDbConnected()) {
+                $result = $response->getResult();
```

Comment:
Would make sense to just save this value to `$results` outside the if, so that the `foreach` loop below can use it too.

Then you could use `if ($last_result = end($results))` and `... = $last_result->getUpdateId();`
(Array will then probably need a `reset($results);` to work with the `foreach` loop)

Question: Could this also just be moved into the loop and updated with each iteration, ending with the last one? Not sure about speed, was just thinking to cut the code a bit.